setup.py

```diff
@@ -176,7 +176,7 @@ def run(self):
     'jupyter_bokeh >=3.0.7',
     'django <4',
     'channels',
-    'pyvista',
+    'pyvista <0.43',  # Breaks an example
```

Comment:
I think https://github.com/holoviz/panel/pull/6020 this fixes it

Reply:
Yep. I just haven't gotten around to updating this PR since the fix.

```suggestion
    'pyvista',  # Breaks an example
```